build.gradle

```diff
@@ -9,10 +9,10 @@ allprojects{
 }
 
 tasks.withType(JavaCompile) {
-    sourceCompatibility = JavaVersion.VERSION_17
+    sourceCompatibility = JavaVersion.VERSION_21
```

Comment:
Had some issues getting to project to behave in IntelliJ related to java 21. This was left behind in java 17.